docs/experience/longtasks.md

```diff
@@ -0,0 +1,65 @@
+title: Long Tasks
+description: Gathering information about long tasks during hard and soft page loads
+
+Experience collects information about long tasks that happen during page loads. According to the RAIL Model, tasks that are running for longer than **50ms** and blocking the main thread are considered long and are recorded by Sematext Cloud. 
```

Comment:
What's RAIL Model?  Should we hyperlink that?

Reply:
Done, linked to explanation.